doc/classes/@GlobalScope.xml

```diff
@@ -176,6 +176,17 @@
 				Returns the point at the given [param t] on a one-dimensional [url=https://en.wikipedia.org/wiki/B%C3%A9zier_curve]Bézier curve[/url] defined by the given [param control_1], [param control_2], and [param end] points.
 			</description>
 		</method>
+		<method name="blend_from_angle">
+			<return type="float" />
+			<param index="0" name="init" type="float" />
+			<param index="1" name="from" type="float" />
+			<param index="2" name="to" type="float" />
+			<param index="3" name="weight" type="float" />
+			<description>
+				Returns blended value of the [param from] and [param to] angles with respect to the [param init], but not beyond the axis inverted [param init]. This method does not respect the shortest path, but instead does not jump values when interpolating more than two interpolations.
```

Comment:
I have no idea what this means yet but I wouldn't want this to be merged with a very apparent typo. This also applies to the other one being exposed.

Also, "_but instead does not jump values_". "instead" compared to what? The first sentence is negative, and the second one is negative too, which reads wrong.
```suggestion
				Returns the blended value of the [param from] and [param to] angles with respect to the [param init], but not beyond the inverted axis [param init]. This method does not respect the shortest path, but instead does not jump values when interpolating more than two interpolations.
```

Reply:
slerp and lerp_angle use the shortest path, but this method does not. The problem of value jumps is explained in the description. For example, multiple uses of lerp_angle would cause this problem, but this method does not.

Reply:
Yes, so mention `slerp` and `lerp_angle`, which are directly related to this method, which has been made to prevent this.